content/en/service_management/service_level_objectives/_index.md

```diff
@@ -51,18 +51,31 @@ Service Level Agreement (SLA)
 Error Budget
 : The allowed amount of unreliability derived from an SLO's target percentage (100% - target percentage) that is meant to be invested into product development.
 
+## SLO Types
+
+When creating SLOs, you can choose from the following types:
+
+**Metric-based SLOs**: can be used for count-based data streams, the SLI is based on the sum of good events divided by the sum of total events.
+
+**Time Slice SLOs or Monitor-based SLOs**: can be be used for time-based data sets, the SLI is based on the amount of time your system exhibits good behavior divided by the total time.
+  - Time Slice SLOs: do not require a Datadog monitor
```

Comment:
I made a couple adjustments to the intro text in https://docs.google.com/document/d/1t6vHaUUHilTgpKhoaA1aUgSn9DMsuCVCEIEiC9A1OJM/edit so this overview here can be updated to match that